### ISSUE TITLE
Can't seem to lint pages via http

### ISSUE THREAD

**Issue by otravers** [twbs/grunt-bootlint#24]
I'm trying to lint a php page generated by WordPress, with the following code:

```
bootlint: {  
    options: {  
        stoponerror: false,
        relaxerror: []
    },
    files: ["http://local.dev/"],
    }
```

I'm removing the doctype from the document at that URL so that it should trigger a Bootlint error, but all I get is "no Bootlint errors" so I don't believe that the page is actually getting scanned. When I point at a sample page that's mapped via the file system (i.e. files: ["path/file.php"]) then Bootlint does trigger errors as expected.

Since I'm using a CMS only linting the rendered pages make sense. (Sorry for having misfiled this earlier under the main bootlint project instead of here, and thanks for doing this!)
